packages/sources/jpegd/CHANGELOG.md

```diff
@@ -0,0 +1,229 @@
+# @chainlink/example-source-adapter
+
+## 1.0.25
+
+### Patch Changes
+
+- Updated dependencies [4d6b8a050]
+  - @chainlink/ea-bootstrap@1.10.4
+  - @chainlink/ea-test-helpers@1.0.1
+
+## 1.0.24
```

Comment:
Can you remove this file since we want to start at 1.0.0 when merging to `develop`?

Reply:
Removed